lib/graphql/query/context.rb

```diff
@@ -53,9 +53,6 @@ def initialize(query:, schema: query.schema, values:)
         @storage = Hash.new { |h, k| h[k] = {} }
         @storage[nil] = @provided_values
         @errors = []
-        @path = []
-        @value = nil
-        @context = self # for SharedMethods TODO delete sharedmethods
```

Comment:
These were leftover from a previous `GraphQL::Query::Context` implementation. I rediscovered them while working on this feature because at first, I thought I was going to need to use `query.context.dup`. I didn't end up doing that (instead, `Query::Context.new` inside `Partial#initialize`) but I'm going to keep these clean-ups.